shim/BUCK

```diff
@@ -74,3 +75,8 @@ execution_platform(
     use_windows_path_separators = host_info().os.is_windows,
     visibility = ["PUBLIC"],
 )
+
+noop_test_toolchain(
```

Comment:
Can you leave a comment here similar to what you said in summary so that in case you dont pick up the followup action item, someone else can